sklearn/manifold/_spectral_embedding.py

```diff
@@ -371,8 +392,9 @@ def spectral_embedding(
             )
             X[:, 0] = dd.ravel()
             X = X.astype(laplacian.dtype)
+            tol = None if eigen_tol == "auto" else eigen_tol
             _, diffusion_map = lobpcg(
-                laplacian, X, tol=1e-5, largest=False, maxiter=2000
+                laplacian, X, tol=tol, largest=False, maxiter=2000
```

Comment:
Setting `tol=None` is technically a change in behavior instead of `1e-5`. Can we list out the estimators and functions in the "Changed Models" section in the whats new?